openedx/core/djangoapps/credit/api.py

```diff
@@ -457,6 +470,7 @@ def get_credit_requirement_status(course_key, username, namespace=None, name=Non
             "namespace": requirement.namespace,
             "name": requirement.name,
             "display_name": requirement.display_name,
+            "order": requirement.order,
```

Comment:
Do we need to expose this through the API?  Since we're returning a list here, the elements already have a defined order -- would it be simpler to just sort them before returning this list?